version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.1.3-SNAPSHOT"
+version in ThisBuild := "0.1.3"
```

Comment:
The version is set by the release plugin, better leave it as it is.

Reply:
Fixed as it was before